PROVISIONING.md

```diff
@@ -161,25 +161,53 @@ Use case: large scale deployments (where automation and security are major drive
 
 ![Provisioning Sequence](architecture/sZTP-sequence.png)
 
+##### Discovery
+
 - Device is powered on
   - Question: how? BMC ? Always on?
 - Device runs DHCP client to send a DHCP request packet to the DHCP server.
   - Question: DHCP client runs from Linux or from UEFI ?
   - Question: Is DHCP can be assumed ? What else mDNS? Static IP ? Mac ?
   - Question: Is DHCP relay required ? in case we are on the different network ?
-- DHCP server assigns an IP address, a default gateway, and the IP address or domain name of the "bootstrap server" to the device.
-  - Question: is "bootstrap server" in the cloud ? local in datacenter ? remote ? vm/container ?
+- DHCP server assigns an IP address, a default gateway, and the IP address or domain name of the "Bootstrap Server" to the device.
+  - Need new custom DHCP option for OPI (similar to SONiC)
+  - Question: is "Bootstrap Server" in the cloud ? local in datacenter ? remote ? vm/container ?
   - Comment: probably need to support multiple preferences by operators. some facilities have to be actually local, other can be proxied to a centralized location, others yet can be completely centralized without a local presence.
-- Device contacts the bootstrap server to get the certificate, bootstrap server will facilitate the request towards CA
-  - Take a look at SCEP (Simple Certificate Enrollment Protocol)
-  - The communication with bootstrap server doesn't have to be secure at this point
-  - Question: preconfigured certificates? how they distributed? what is alternative to certificates? PKI based crypto ?
-- If we want to protect stolen/mistaken shiipment (device needs to authenticate network) we have to use Vouchers
-  - Take a look at RFC 8366 - A Voucher Artifact for Bootstrapping Protocols
+- Need to consider mDNS as another option if DHCP is absent.
+- Need to consider LLDP/SSDP as well.
+
+##### Network trusts DPU
```

Comment:
DPU or IPU

Reply:
For environments that are predominately IPv6 based like mine, we'll need a solution that suppots SLAAC.

Reply:
@mebagwell I think we have to separate the Discovery and Address assignment from Provisioning into 2 pluggable modules, then we can have flexibility to run in several environments... right ?